editions/pluginlibrary/tiddlers/GettingStarted.tid

```diff
@@ -0,0 +1,11 @@
+title: GettingStarted
+
+This server let's you test the plugin import mechanism. 
```

Comment:
"let's" shouldn't have an apostrophe. Clearer to say:

This edition contains the components needed to set up a local plugin library for testing or development.